Signal/src/call/PeerConnectionClient.swift

```diff
@@ -167,13 +193,19 @@ class PeerConnectionClient: NSObject, RTCPeerConnectionDelegate, RTCDataChannelD
     }
 
     deinit {
-        Logger.debug("[PeerConnectionClient] deinit")
+        // TODO: We can demote this log level to debug once we're confident that
+        // this class is always deallocated.
+        Logger.info("[PeerConnectionClient] deinit")
```

Comment:
Note: there are still cases where `PeerConnectionClient` isn't always deallocated.  I'll try to address this in a future PR.